CS_Ed_Week/CS_Ed_Week.md

```diff
@@ -12,152 +15,186 @@ A workshop for getting young people interested in computer science
 - Both arrays and linked lists have their weaknesses:
-    - Since arrays are one continuous block of elements, reading a particular element is just a matter of knowing its index. Insertions however are much slower, since you have to move all of the elements to the right of it over 1.
-    - Reading an element of a linked list takes longer, because you do not know a particular elements position. You instead have to start at the beginning of the list, which will tell you the address of the next element and the next, until you find what you are looking for. Insertion however, are much faster, since you just have to change where one element is pointing to.
-    - Deletions have the same characteristics as as insertions in this context
+  - Since arrays are one continuous block of elements, reading a particular element is just a matter of knowing its index. Insertions however are much slower, since you have to move all of the elements to the right of it over 1.
+  - Reading an element of a linked list takes longer, because you do not know a particular element's position. You instead have to start at the beginning of the list, which will tell you the address of the next element and the next, until you find what you are looking for. Insertions however, are much faster, since you just have to change where one element is pointing to.
+  - Deletions have the same characteristics as as insertions in this context.
 
 ## Part X: Lists
+
 ### Creating lists
+
 - A sytax for creating a list is as follows:
-    - `list_name = [1, 2, 3, 4, 5]`
-    - This list has 6 elements
+  - `list_name = [1, 2, 3, 4, 5]`
+  - This list has 6 elements.
 - It is possible to mix and match data types in a list.
-- Each element in a list has a position, and that position is referred to as its index
+- Each element in a list has a position, and that position is referred to as its index.
 - Indices start at zero, so on the list above `list_name[0]` is 1, and `list_name[4]` is 5.
+
 ### Manipulating Lists
-- Python provide a rich set of list manipulation methods
-- A method is a function. To call a method on a variable, you use what's known as dot notation
+
+- Python provides a rich set of list manipulation methods.
+- A method is a function. To call a method on a variable, you use what's known as dot notation.
 - Here are some common list operations:
-    - Appending a list element: `list_name.append(6)`
-      - list_name -> [1, 2, 3, 4, 5, 6]
-    - Popping (removing the last element): `popped_item = list_name.pop()`
-      - list_name -> [1, 2, 3, 4, 5]
+  - Appending a list element: `list_name.append(6)`
+    - list_name -> [1, 2, 3, 4, 5, 6]
+  - Popping (removing the last element): `popped_item = list_name.pop()`
+    - list_name -> [1, 2, 3, 4, 5]
       - popped_item -> 6
```

Comment:
I think this line should have the same nesting level as the previous, since both variables relate to line `popped_item = list_name.pop()`

Reply:
Great catch, I'm going to attempt the suggestion feature.

```suggestion
    - popped_item -> 6
```